### PR TITLE
{astro}[foss/2019a] CRPropa v3.1.5 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by jose-d**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+sources = ['%(version)s.tar.gz']
+checksums = ['ee394b42967fabe93b83205751a38b70cc67a126bce5968d49656a556ff022d5']
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('HDF5', '1.10.5'),
+    ('SciPy-bundle', '2019.03'),
+]
+
+builddependencies = [
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb]: Please put builddeps above dependencies, just for style.

File: easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb]: This is probably easier to implement as a CMakePythonPackage.
You can then remove the modextrapaths and the ".../site-packages" from sanity_check.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/37855aa65da2dda0b9f61670d89cc6c4 for a full test report.

**Comment by jose-d** [comment]
Test report by @jose-d
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
koios2 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/539465920481c327df0ebb67362c3e33 for a full test report.

File: easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb (hunk 2 above)
Comment (jose-d) [reply easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb]: done.

**Comment by akesandgren** [comment]
Going in, thanks @jose-d!

**Pull request merged**